docs/content/tools/xdebug.md

```diff
@@ -4,210 +4,182 @@ aliases:
   - /en/master/tools/xdebug/
 ---
 
-
 Xdebug can be used to debug both web requests as well as cli scripts (e.g., Drush commands).
 
-## Setup
+## Stack Setup {#setup}
 
-`xdebug` extension is disabled by default since it causes about a 20% performance hit. To enable it:
+Xdebug integration is disabled by default as it causes a roughly 20% performance hit. To enable it:
 
 ```bash
-fin config set XDEBUG_ENABLED=1
+fin config set --env=local XDEBUG_ENABLED=1
 fin project start
 ``` 
 
-To verify that Xdebug was enabled run:
+To verify that Xdebug was enabled:
 
 ```bash
 $ fin exec php -v | grep -i xdebug
-    with Xdebug v2.5.1, Copyright (c) 2002-2017, by Derick Rethans
+    with Xdebug v2.7.2, Copyright (c) 2002-2019, by Derick Rethans
 ```
 
-Note: Starting with Docksal v1.6.0 (and assuming the default stack is used), installing the companion browser extension is no longer necessary. Once Xdebug is enabled, debugging sessions will be started automatically.
-
-### Debugging Drush 8
-
-To debug Drush 8 commands, a path mapping will need to be made for the internal drush binary and the IDE. We recommend copying the drush binary to your local project and using file mapping.
-
-This can be done by running the following in the project root:
-
-```
-$ mkdir bin
-$ fin exec cp /usr/local/bin/drush8 ./bin/drush
-```
+{{% notice note %}}
+Starting with Docksal v1.6.0 (and assuming the default stack is used), installing the companion browser extension 
+is no longer necessary. Once Xdebug is enabled, debugging sessions will be started automatically.
+{{% /notice %}}
```

Comment:
I guess we can drop this note already

Reply:
ok, I was thinking about this too. Will drop.